src/pybind/mgr/dashboard/frontend/e2e/block/mirroring.e2e-spec.ts

```diff
@@ -30,4 +32,32 @@ describe('Mirroring page', () => {
       expect(mirroring.getTabText(2)).toEqual('Ready');
     });
   });
+  describe('checks that edit mode functionality shows in the pools table', () => {
+    const poolName = 'mirrorpoolrq';
+    beforeAll(() => {
+      pools.navigateTo('create'); // Need pool for mirroring testing
+      pools.create(poolName, 8, 'rbd').then(() => {
```

Comment:
The reasoning for including 'rbd' is because the option in the applications table is called 'rbd'. Since a by.cssContainingText() is used to click said option, renaming it wouldn't allow for the proper application to be chosen.

Reply:
Sorry, my bad - I mend the pool name. Have done to many reviews these days :P But that looks OK.